lib/semian/lru_hash.rb

```diff
@@ -5,8 +5,6 @@ class LRUHash
   # everytime we set a new resource. A default window of
   # 5 minutes will allow empty item to stay in the hash
   # for a maximum of 5 minutes
-  extend Forwardable
-  def_delegators :@table, :size, :count, :empty?, :values
   attr_reader :table
```

Comment:
This is suspicious for thread safety. One could still grab @table with this and do dangerous stuff. Do we need to maintain the public API for this class?

Reply:
Yeah, we should remove that method. I'll do that.